build.gradle.kts

```diff
@@ -4,6 +4,15 @@ plugins {
     alias(libs.plugins.kotlin.jvm) apply false
     alias(libs.plugins.checksum) apply false
     alias(libs.plugins.shadow) apply false
+    alias(libs.plugins.kotlinx.binary.compatibiltiy.validator)
+}
+
+apiValidation {
+    ignoredProjects += setOf(
+        "ktlint-api-consumer",
+        "ktlint-bom",
+        "ktlint-ruleset-template",
```

Comment:
I forgot to remove `ktlint-test-ruleset-provider-v2-deprecated` from ktlint-bom when dropping the module. If it is removed, would it then be possible to share this configuration so that we only have one list of modules to be excluded?

Reply:
>If it is removed, would it then be possible to share this configuration so that we only have one list of modules to be excluded?

Should be done in [`ecba7f0` (#2131)](https://github.com/pinterest/ktlint/pull/2131/commits/ecba7f0cb6b0891b69771c113bea1958723e7460) ✅ 
The list is defined once (I tried to be extra explicit with the shared variable name) and I modified the `ktlint-bom` config to preserve existing behavior (I'm appending the legacy module manually, it seems to be ignored but seeing the project's code is still in the repo I'm trying to avoid making any extra changes).

Reply:
Nice solution and thanks for careful double check. Indeed the module is still in the repository, because I still have one [unit test](https://github.com/pinterest/ktlint/blob/4de7cab9dcc1b19d1167b97fae26f6000453cd9c/ktlint-cli/src/test/kotlin/com/pinterest/ktlint/cli/RuleSetsLoaderCLITest.kt#L14) that needs a jar with a v2-provider.